src/app/checkout/CheckoutStepHeader.tsx

```diff
@@ -40,41 +36,35 @@ const CheckoutStepHeader: FunctionComponent<CheckoutStepHeaderProps> = ({
             ) }
             onClick={ preventDefault(isEditable && onEdit ? () => onEdit(type) : noop) }
         >
-            <div className="stepHeader-row">
-                <div className="stepHeader-figure stepHeader-column">
-                    <IconCheck
-                        additionalClassName={ classNames(
-                            'stepHeader-counter',
-                            'optimizedCheckout-step',
-                            { 'stepHeader-counter--complete': isComplete }
-                        ) }
-                    />
+            <div className="stepHeader-figure stepHeader-column">
```

Comment:
Do we need to have the `stepHeader-row` wrapper div?

Reply:
We don't need it anymore, because we moved suggestion block from wrapped stepHeader flex div.

We needed stepHeader-row to make 2 rows:
1) For main stepHeader content;
2) For suggestion block;